apps/script/HoC2023AiGenerateWeights.py

```diff
@@ -23,7 +23,7 @@
 color_palettes_map = {
   'rainbow pastel': 'default', # Light
   'green blue': 'cool', # Cool
-  'electronic': 'elecronic', # Electronic
+  'electronic': 'electronic', # Electronic
```

Comment:
Thank you!

Reply:
Next task is to write a function in this file to pull names from block config file.